repositories/patches/geometry2_remove_rclcpp_components.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/tf2_ros/src/static_transform_broadcaster_node.cpp b/tf2_ros/src/static_transform_broadcaster_node.cpp
+index 37f2cf7..c536fd1 100644
+--- a/tf2_ros/src/static_transform_broadcaster_node.cpp
++++ b/tf2_ros/src/static_transform_broadcaster_node.cpp
+@@ -98,7 +98,3 @@ StaticTransformBroadcasterNode::StaticTransformBroadcasterNode(const rclcpp::Nod
+   broadcaster_->sendTransform(tf_msg);
+ }
+ }  // namespace tf2_ros
+-
+-#include "rclcpp_components/register_node_macro.hpp"
+-
+-RCLCPP_COMPONENTS_REGISTER_NODE(tf2_ros::StaticTransformBroadcasterNode)
```

Comment:
Just for my understanding, why is this needed?

Reply:
`register_node_macro.hpp` is from `rclcpp_components` which depends on `ament_index` which isn't needed for bazel rules. `ament_index` is also removed in `rclcpp`.